erpnext/public/js/controllers/taxes_and_totals.js

```diff
@@ -754,5 +754,61 @@ erpnext.taxes_and_totals = erpnext.payments.extend({
 				}
 			});
 		})
+	},
+	set_and_update_cultivation_tax: function () {
+		if (["Sales Order", "Sales Invoice", "Delivery Note", "Stock Entry"].includes(this.frm.doctype)) {
+			if (!this.frm.doc.license) {
```

Comment:
This check should happen in the selling controller instead, before this function is called